logger_core/src/lib.rs

```diff
@@ -184,8 +208,10 @@ pub fn init(minimal_level: Option<Level>, file_name: Option<&str>) -> Level {
                 });
         }
         Some(file) => {
-            let file_appender =
-                LazyRollingFileAppender::new(Rotation::HOURLY, FILE_DIRECTORY, file);
+            // Check if the environment variable GLIDE_LOG is set
+            let logs_dir =
+                directory_from_env(ENV_GLIDE_LOG_DIR).unwrap_or(FILE_DIRECTORY.to_string());
```

Comment:
```suggestion
                directory_from_env_or_constant();
```

Reply:
I prefer to explicit Rust way of `unwrap_or(..)`